src/indicator_cumulative_number.Rmd

```diff
@@ -69,3 +72,70 @@ About point 7 of [#21](https://github.com/trias-project/pipeline/issues/21), ple
 
 see GitHub issue [#20](https://github.com/trias-project/pipeline/issues/20)
 
+```{r Determine presence per species(key) per year}
+
+unique_df <- data %>%
+ filter(!is.na(startDate)) %>%
+ distinct(key, startDate, endDate)
+
+# Create yearslist using startDate and endDate 
+counts_ias <- data.frame(year = c(min(unique_df$startDate, na.rm = TRUE):
+                                 max(unique_df$endDate, na.rm = TRUE))) %>%
+ as_tibble()
+
+count_presence <- function(year, df) {
+ df %>% filter(startDate <= year & endDate >= year) %>%
+   nrow() %>% as_tibble() %>% rename(counts = value)
+}
+
+counts_ias %<>%
+ rowwise() %>%
+ do_(interp(~ count_presence(year = .$year, df = unique_df))) %>%
+ bind_cols(counts_ias) %>% select(year, everything())
+```
+
+```{r Calculate time sections}
+
+counts_ias$decade <- counts_ias$year - counts_ias$year %% 10
+
+
+```
+
+
+
+```{r Basic english Graphs}
+
+counts_ias %>% ggplot( aes(x = year, 
+                            y = counts)) +
+                geom_line() +
+                xlab("Year") +
+                ylab("Number of alien species (cumulative sum)") +
+                theme(strip.text.y = element_text(size = 16)) +
+                scale_x_discrete(labels = seq(1200, 2030, 15)) + 
+                scale_x_continuous(breaks = seq(1200, 2010, 50)) +
+                theme_inbo(base_size = 14)
+
+counts_ias %>% ggplot( aes(x = year, 
+                            y = counts)) +
+                geom_bar(stat = "identity") +
+                xlab("Year") +
+                ylab("Number of alien species (cumulative sum)") +
+                theme(strip.text.y = element_text(size = 16)) +
+                scale_x_discrete(labels = seq(1200, 2030, 15)) + 
```

Comment:
why not using the mindate and maxdate? Any reason for this. Seems strange to hardcode these years